scala-package/mxnet-demo/src/main/scala/sample/ImageClassificationExample.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sample
+
+import org.apache.mxnet.{Context, DType, DataDesc, Shape}
+import org.kohsuke.args4j.{CmdLineParser, Option}
+import org.slf4j.LoggerFactory
+import org.apache.mxnet.infer.{ImageClassifier, _}
+
+import scala.collection.JavaConverters._
+import java.io.File
+import java.net.URL
+import org.apache.commons.io._
+
+import scala.collection.mutable.ListBuffer
+
+/**
+  * Example showing usage of Infer package to do inference on resnet-152 model
+  * Follow instructions in README.md to run this example.
+  */
+object ImageClassificationExample {
+
+  def downloadImage() : String = {
```

Comment:
can you create one downloadUrl method instead of separate ?, we can move this to a TestUtils method and add hash check etc., later
downloadUrl(downloadFrom, downloadTo)